server/src/Infrastructure/DependencyInjection.cs

```diff
@@ -21,7 +21,11 @@ public static IServiceCollection AddInfrastructure(this IServiceCollection servi
             throw new InvalidOperationException("The JWT signing key must be configured.");
         }
 
-        services.AddDbContext<ApplicationDbContext>(options => options.UseSqlServer(configuration.GetConnectionString("DefaultConnection")));
+        services.AddDbContext<ApplicationDbContext>(options =>
+            {
+                options.UseSqlServer(configuration.GetConnectionString("DefaultConnection"));
+                options.AddInterceptors(new AuditableEntityInterceptor()); 
+            });    
```

Comment:
```suggestion
        services.AddDbContext<ApplicationDbContext>(options =>
        options.UseSqlServer(configuration.GetConnectionString("DefaultConnection"))
        .AddInterceptors(new AuditableEntityInterceptor()));
```

Reply:
no need to reference options multiple times

Reply:
Agree